interop-testing/src/test/java/io/grpc/testing/integration/Http2OkHttpTest.java

```diff
@@ -88,8 +88,7 @@ protected ManagedChannel createChannel() {
         .overrideAuthority(GrpcUtil.authorityFromHostAndPort(
             TestUtils.TEST_SERVER_HOST, serverPort));
     try {
-      builder.sslSocketFactory(TestUtils.newSslSocketFactoryForCa(
-          TestUtils.loadCert("ca.pem")));
+      builder.sslSocketFactory(TestUtils.newSslSocketFactoryForCa(TestUtils.loadCert("ca.pem")));
```

Comment:
Were you planning on changing this at all? Nobody is using the new method, was there some other intended user?